src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPCompletionProposal.java

```diff
@@ -116,7 +132,12 @@ public void handleInsert(@NotNull InsertionContext context) {
         // Apply all text edits
         apply(context.getDocument(), context.getOffset(CompletionInitializationContext.SELECTION_END_OFFSET));
 
-        if (shouldStartTemplate(template)) {
+        // Just move to the first tab stop if there's a single invocation argument with no default value
+        if (shouldMoveToFirstTabStop(template)) {
```

Comment:
Now we only move to the first tab stop for a template with a single non-end variable that has no value. That happens independent of client config because it doesn't make sense to start an editor template for a single tab stop with no default value. Instead just place the caret in that same location and let the user type what they need.